ci/resources.yml

```diff
@@ -3,16 +3,17 @@ resources:
   type: git
   source:
     branch: master
-    private_key: ((svc-dip.private_key))
-    uri: git@github.ucds.io:dip/aws-management-infrastructure.git
+    uri: https://((dataworks.enterprise_github_url))/dip/aws-management-infrastructure.git
+    username: ((dataworks.enterprise_github_username))
+    password: ((dataworks-secrets.enterprise_github_pat))
   check_every: 720h
-  webhook_token: ((ucfs-github.webhook-token))
+  webhook_token: ((dataworks.concourse_github_webhook_token))
 - name: acm-pca-cert-generator
   type: github-release
   source:
     owner: dwp
     repository: acm-pca-cert-generator
-    access_token: ((github_com.token))
+    access_token: ((dataworks-secrets.concourse_github_pat))
   check_every: 3m
```

Comment:
can you add a webhook to the acm-pca repo and change this check time to 720h?

Reply:
Thanks, done